File: Python Programming/problems on pattern printing/problem4.py
'''
4.Write a program which accept string from user and print below pattern. 
	
	Input :   “Marvellous” 
	
	Output :   M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S     
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S 
	

	Author : Milind Nalawade

	Date : 	07/08/2021
	
	Language : Python 3.9.0
	
'''
#====
#Logic
#====
def Output(arr):
	no=len(arr)
	print("\n Output : \n\n")
	while no:
		print("	",end="")
		for i in range(len(arr)):
			print(arr[i].upper(),end=" ")
			i+=1
		i=0
		print("\n",end="")
		no-=1
		
#=============
#Entry point function
#=============
def main():
	arr=input("\n Enter your string : ")
	Output(arr)
	
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C Programming/problems on pattern printing/problem4.c
/*
4.Write a program which accept string from user and print below pattern. 
	
	Input :   “Marvellous” 
	
	Output :   M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S     
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S    
					M A R V E L L O U S 
	

	Author : Milind Nalawade

	Date : 	07/08/2021
	
	Language : C programming
	
*/

#include<stdio.h>
#include<string.h>

//Logic 
void Output(char arr[])
{
	int i=0,j=0;
	printf("\n Output : \n\n");
	while(j!=strlen(arr))
	{
		printf("	");
		while(i!=strlen(arr))
		{
			
			if (arr[i]>=97 && arr[i]<=122)
			{
				printf("%c ",arr[i]-32);
			}
			else
			{
				printf("%c ",arr[i]);
			}	
			i++;
		}
		i=0;
		printf("\n");
		j++;
	}
}

//Entry point function
int main()
{
	char arr[30];
	printf("\n Enter your string : ");
	scanf("%[^'\n']s",arr);
	Output(arr);
	return 0;
}	